azure/packages/azure-local-service/package.json

```diff
@@ -52,7 +52,7 @@
 		"typescript": "~4.5.5"
 	},
 	"engines": {
-		"node": ">=14.0.0"
```

Comment:
This package is one we actually publish, so changing this will break users of it, not just suggest that developers upgrade. I'd recommend not changing this for now, and when we do change it, do it on next and announce it to consumers in advance.

Reply:
I will revert this (and other package.json files) but keep the root package.json in this change, that one shouldn't affect users of the package.